config/manager/kustomization.yaml

```diff
@@ -1,6 +1,5 @@
 resources:
 - manager.yaml
-- clustermanagementaddon.yaml
```

Comment:
Looking in the Makefile, I find three invocations of kustomize to build config/default: in the targets `chart`, `deploy`, and `undeploy`.